### PR TITLE
Add PL lang

### PR DESCRIPTION
**Pull request by Gari7572**

### CODE REVIEW

**Comment by getneil** [comment]
@Gari7572 please import it also in the `svelte/src/libs/translations/index.ts` after ptbr. thanks

File: svelte/src/libs/translations/index.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ import ru from "./languages/ru.json";
 import uk from "./languages/uk.json";
 import zh from "./languages/zh.json";
 import ptbr from "./languages/ptbr.json";
+import pl from "./language/pl.json";
```
Comment (getneil) [inline svelte/src/libs/translations/index.ts:12]: @Gari7572 misssing an s

you can also test the ossapp locally by just executing 
```
xc setup

then

xc dev
```

assuming you pkgx cli https://pkgx.sh/

**Comment by Gari7572** [comment]
Corrected, glad to be useful to the project. The rules haven’t changed, will I now get a role as a contributor on discord?

**Comment by mfts** [comment]
@Gari7572 yes that's correct. Once the PR is merged, you'll receive the contributor role in Discord. Please share your discord handle with me so it can be assigned properly.

**Comment by Gari7572** [comment]
https://discordapp.com/users/892266604627386418/ Thank you very much if you remember that you once helped me, but it didn’t work out for me, I still forced myself to do it.

**Comment by Gari7572** [comment]
This was my first experience, together with the Tea project I am learning and blossoming

**Pull request merged**